modules/benchmark/R/match.timestep.R

```diff
@@ -6,9 +6,9 @@
 ##' @export match.timestep
 ##' 
 ##' @author Istem Fer
-match.timestep <- function(date_coarse, date_fine, data_fine) {
+match.timestep <- function(date.coarse, date.fine, data.fine) {
   
-  midpoints <- c(-Inf, head(date_fine, -1)) + c(0, diff(as.numeric(date_fine)) / 2)
+  midpoints <- c(-Inf, head(as.numeric(date.fine), -1)) + c(0, diff(as.numeric(date.fine)) / 2)
   
-  return(data_fine[findInterval(date_coarse, midpoints)])
+  return(data.fine[unique(findInterval(date.coarse, midpoints))])
```

Comment:
Not sure I see why you need a unique here. That seems to risk changing the length and order of the data